### ISSUE TITLE
fix `fly.watch` test

### ISSUE THREAD

**Issue by lukeed** [lukeed/taskr#205]
Currently [removed](https://github.com/flyjs/fly/blob/master/test/fly.js#L95) since it never finished. The two assertions [within the fly.host hijack](https://github.com/flyjs/fly/blob/master/test/fly.js#L121) never happen.

This test had to be removed after removing the `process.chdir` side effect (#202).

**Comment by lukeed** [comment]
No longer relevant after #218
